packages/components/releaseNotes/labkey/components.md

```diff
@@ -1,6 +1,11 @@
 # @labkey/components
 Components, models, actions, and utility functions for LabKey applications and pages.
 
+### version TBH
+*Released*: TBD
+* Issue 39819: List designer: small nits
```

Comment:
Might be good to put a little more info into this text then "small nits" related to what the issues are that were fixed.

Reply:
Good point, updated!